docs/community/glossary.md

```diff
@@ -0,0 +1,15 @@
+| channel            | A channel is a grayscale image. In fluorescence microscopy,  each channel corresponds to a range of wavelengths. An RGB image from digital camera consists of 3 channels - red, green, blue.                                                                                                                                                |
+| color map          | An array of colors used to map pixel data to the actual color values.                                                                                                                                                                                                                                                                       |
+| dock widget        | A widget that can be attached to the main napari viewer at a specified position.                                                                                                                                                                                                                                                            |
+| hook               | Hook is the place within napari where we “promise” to call functions created by external developers and installed by the user.                                                                                                                                                                                                              |
+| hook specification | A function signature that defines the API that a plugin developer must adhere to when writing the function that we promise to call somewhere in the napari codebase.                                                                                                                                                                        |
+| keybinding         | The key or key combination that causes some functions to occur.                                                                                                                                                                                                                                                                             |
+| labels             | The labels layer is an image where pixel values are integers. Each unique integer represents a unique object, feature or classification.                                                                                                                                                                                                    |
+| layer              | Layers are the viewable objects that can be added to napari viewer. Currently napari supports the following layers: image, labels, points, shapes, surface, tracks, vectors.                                                                                                                                                                |
```

Comment:
* Would "viewable data" be clearer than "viewable objects"? Interested to hear what other people think.

Reply:
I think the word data should be in there somewhere, yes. Maybe 

"Layers are the objects through which your data can be added to the napari viewer".

Each of the individual layers should probably link to the layer docs.

Reply:
"Layers are the viewable objects that can be added to a viewer." is copied from API reference > napari.layers.
I find "viewable object" more precise than "viewable data", because layers can contain some objects that do not represent data - for example, use shape layer to draw a line to get the intensity profile. 

Yes hyperlinks should be added :)

If we do want to change its definition, we may want to make sure it is consistent across the documentation.